Drop-In/src/main/java/com/braintreepayments/api/dropin/DropInRequest.java

```diff
@@ -63,6 +64,16 @@ public DropInRequest tokenizationKey(String tokenizationKey) {
     }
 
     /**
+     * This method is optional. Amount is only used for 3D Secure verifications.
+     *
+     * @param amount Amount of the transaction.
```

Comment:
Does the optional tag here indicate that it's optional for drop-in, or optional for 3DS? My understanding was that an amount was required for 3DS.